src/systemctl/systemctl-show.c

```diff
@@ -2095,6 +2102,7 @@ static int show_one(
                 .startup_memory_zswap_max = CGROUP_LIMIT_MAX,
                 .memory_limit = CGROUP_LIMIT_MAX,
                 .memory_peak = CGROUP_LIMIT_MAX,
+                .memory_swap_peak = CGROUP_LIMIT_MAX,
```

Comment:
UINT64_MAX ?

Reply:
All other fields before go with `CGROUP_LIMIT_MAX`, which is a define to `UINT64_MAX`. So I assume its better to keep with the existing pattern. Or do you still want me to change it?

Reply:
If these are equivalent, then the current form is ok.